wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -14,15 +14,19 @@
     - `action_icon` - icon for the 'action' button, default is 'icon-plus'
 
 {% endcomment %}
-<header class="header {% if merged %}merged{% endif %} {% if search_form %}hasform{% endif %}">
+<header class="header{% if merged %} merged{% endif %}{% if search_form %}hasform{% endif %} {{ classname }}">
```

Comment:
Maybe avoid this unless you need it,
But if you do think we need this ability to add a classname please ensure you also add it to the description above (variables accepted by this template).